sources/iTermAdvancedSettingsModel.m

```diff
@@ -134,6 +134,8 @@ + (NSString *)name { \
 DEFINE_BOOL(indicateBellsInDockBadgeLabel, YES, @"General: Indicate the number of bells rung while the app is inactive in the dock icon’s badge label");
 DEFINE_STRING(downloadsDirectory, @"", @"General: Downloads folder.\nIf set, downloaded files go to this location instead of the user’s $HOME/Downloads folder.");
 DEFINE_FLOAT(pointSizeOfTimeStamp, 10, @"General: Point size for timestamps");
+DEFINE_INT(terminalMargin, 5, @"General: Terminal Margin\nHow much space to leave between the left and right edges of the terminal.");
```

Comment:
Add this text to both terminalMargin and terminalVMargin at the end of the description, please:

You must restart iTerm2 after modifying this property.